css/_sidenavs.scss

```diff
@@ -110,3 +110,14 @@
     margin-right: 10px!important;
     margin-left: 5px!important;
 }
+
+md-sidenav {
```

Comment:
Why not. 
If you prefer not to add a new class, is it possible to override the built-in angular material classes?